mithril-common/src/digesters/dummy_immutable_db_builder.rs

```diff
@@ -156,3 +215,12 @@ fn write_dummy_file(optional_size: Option<u64>, dir: &Path, filename: &str) -> P
 
     file
 }
+
+fn get_test_dir(subdir_name: &str) -> PathBuf {
+    TempDir::create(subdir_name, LEDGER_DIR);
+    TempDir::create(subdir_name, VOLATILE_DIR);
+    let immutable_dir = TempDir::create(subdir_name, IMMUTABLE_DIR);
+    let test_dir = immutable_dir.parent().unwrap().to_path_buf();
+
+    test_dir
```

Comment:
It's better to only use the `TempDir` api to create the parent dir, else you have no guarantee that all three folders share a common parent:
```suggestion
    let db_dir = TempDir::create("test_cardano_db", subdir_name);
    for subdir_name in [LEDGER_DIR, IMMUTABLE_DIR, VOLATILE_DIR] {
        fs::create_dir(db_dir.join(subdir_name)).unwrap();
    }

    db_dir
```

Reply:
OK, thanks, I hadn't noticed that. I'll make the change.

Reply:
Done ✅